packages/react-components/react-utilities/src/virtualParent/getParent.ts

```diff
@@ -36,3 +36,23 @@ export function getParent(child: Node | null, options: GetParentOptions = {}): N
 
   return child?.parentNode || null;
 }
+
+export function getParentInShadowDOM(child: Node | null, options: GetParentOptions = {}): Node | null {
```

Comment:
We need cypress tests to make sure that this works.

Reply:
I've added ShadowDOM tests for the consolidated getParent().